spec/features/admin_series_create_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+describe 'the admin user flow', type: :feature do
+
+  let(:user)  {User.create(email_address: "sam@example.com", admin: true, password: "password")}
+  let(:subscription)  {Subscription.create(email_address: "coll@example.com", name: "Colleen")}
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Space missing inside {.<br>Space missing inside }.